source/application/SXRMB/SXRMBAppController.cpp

```diff
@@ -494,9 +494,7 @@ void SXRMBAppController::onShowAmbiantSampleStageMotorsTriggered()
 	motorGroupView->setMotorGroupView(motorGroupName);
 	motorGroupView->showAvailableMotorGroupChoices(false);
 
-	AMDialog *showMotorDialog = new AMDialog(motorGroupName);
-	showMotorDialog->layoutDialogContent(motorGroupView);
-	showMotorDialog->exec();
+	motorGroupView->show();
```

Comment:
@XuetaoLiu-CLS, you would be better off using ```QWidget::raise()``` rather than ```show()```.  If you only use show the window can get lost and if you trigger the view again it will still be hidden behind other windows.  Using raise will always bring it to the forefront.